screens/resources/ResourceScreen.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { TouchableOpacity, Button, View } from 'react-native';
+import { Linking } from 'react-native';
+import { TopText } from '../../styled/resources';
+
+const resources = [
+  {
+    name: 'DC Central Kitchen',
+    url: 'https://dccentralkitchen.org/'
+  },
+  {
+    name: 'Healthy Corners Initiative',
+    url: 'https://dccentralkitchen.org/healthy-corners/'
+  },
+  {
+    name: 'Important Link',
+    url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ'
+  }
+];
+
+class ResourceScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <View>
+        <TouchableOpacity
```

Comment:
You probably want to put this in at least a styles object, so that it's easily exported to a stylesheet and then can be refactored to `styled-components`. Since the Resources screen is going to be more complex than we thought, I think it's worth it to pull this out and add it to the `styled/` folder - you can make it `resources.js` (-:

Reply:
Given this week's Figma and our latest idea of the Resources page, I think it would be a good idea to create a new component - possibly something like `ResourceCard` - in addition to the `styled-components` in `resources.js`! The main reason is that all the resources links are in the format (style + content-wise).